default_app/main.js

```diff
@@ -142,6 +128,9 @@ function startRepl () {
     process.exit(1)
   }
 
+  // prevent quitting
+  app.on('window-all-closed', () => {})
```

Comment:
wait... this doesn't seem right? I don't think we want to prevent quitting in the default app?

Reply:
This is REPL mode, it's supposed to keep running after all windows are closed.

Reply:
It was handled by the `option.interactive` check before